src/Microsoft.AspNet.Razor.Runtime/TagHelpers/TagHelperDescriptorFactory.cs

```diff
@@ -671,6 +673,7 @@ private static bool ShouldSkipDescriptorCreation(bool designTime, IMemberInfo me
             string attributeName,
             string typeName,
             bool isIndexer,
+            bool isStringProperty,
```

Comment:
Do we still need to take in the `isStringProperty` here? Doesn't it get calculated via `typeName` correctly for these tests?